cobc/warning.def

```diff
@@ -114,5 +114,8 @@ CB_ONWARNDEF (cb_warn_dialect, "dialect",
 CB_NOWARNDEF (cb_warn_source_after_code, "dangling-text",
 	_("  -Wdangling-text       warn about source text after program-area"))
 
+CB_ONWARNDEF (cb_missing_newline, "missing-newline",
```

Comment:
This one (the separate warning flag) can directly go into 3.x. As copyright assignments are done, you could do it directly, too (I just need your SF user to provide write access).